packages/core/documentation/src/components/ProductDocumentModal.vue

```diff
@@ -0,0 +1,393 @@
+<template>
+  <Teleport
+    to="#kong-ui-app-layout-teleport-default-slot"
+  >
+    <KPrompt
+      class="edit-document-modal"
+      data-testid="edit-document-modal"
+      is-visible
+      @canceled="handleClickCancel"
+      @proceed="handleClickSave"
+    >
+      <template #header-content>
+        <div class="title">
+          {{ titleText }}
+          <div class="subtitle">
+            {{ i18n.t('documentation.form_modal.subtitle') }}
+            <br>
+            <KExternalLink
+              class="doc-link"
+              :href="externalLinks.serviceDocumentationDocs"
+            >
+              {{ i18n.t('documentation.form_modal.documentation_link_text') }}
+            </KExternalLink>
+          </div>
+        </div>
+      </template>
+      <template #body-content>
+        <KAlert
+          v-if="errorMessage"
+          :alert-message="errorMessage"
+          appearance="danger"
+          class="bottom-spacing"
+        />
+
+        <!-- TODO: help tooltip-->
```

Comment:
Does this still need to be done?

Reply:
don't think so, it was left over from the existing component